server/cmwell-dc/src/pack/logback.xml

```diff
@@ -169,6 +169,30 @@
         <discardingThreshold>0</discardingThreshold>
     </appender>
 
+    <appender name="BAD_DATA" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>logs/bad-data.log</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+            <!-- daily rollover with compression -->
+            <fileNamePattern>logs/bad-data-%d{yyyy-MM-dd}.%i.gz</fileNamePattern>
+            <!-- keep 1 week worth of history (max 20GB) with each file compressed after 100MB -->
+            <maxFileSize>100MB</maxFileSize>
+            <maxHistory>7</maxHistory>
+            <totalSizeCap>20GB</totalSizeCap>
+            <!-- in case of process terminating too early for rollover - do the rollover during start -->
+            <cleanHistoryOnStart>true</cleanHistoryOnStart>
+        </rollingPolicy>
+        <encoder>
+            <pattern>%date{ISO8601} %-44.44([%thread]) %-5level %-36.36logger{36} - %msg%n</pattern>
+        </encoder>
+        <!-- false quadruples logging throughput -->
+        <immediateFlush>true</immediateFlush>
+    </appender>
+
+    <appender name="ASYNC_BAD_DATA" class="ch.qos.logback.classic.AsyncAppender">
+        <appender-ref ref="BAD_DATA" />
```

Comment:
same argument - where is the maxFlushTime?

Reply:
Done